packages/fluentui/react-northstar/src/components/Dialog/Dialog.tsx

```diff
@@ -258,10 +258,24 @@ export const Dialog = (React.forwardRef<HTMLDivElement, DialogProps>((props, ref
     },
   });
 
+  // when press left click on Dialog content and hold, and mouse up on Dialog overlay, Dialog should keep open
+  const isMouseDownInsideContent = React.useRef(false);
+  const registerMouseDownOnDialogContent = (e: React.MouseEvent) => {
```

Comment:
this should be testable in cypress by triggering a mousedown followed by a click somewhere else ?

Reply:
Heyy, if I do 
```
cy.trigger('mousedown').
trigger('mousemove', {clientX: 0, clientY: 0}). // move mouse to top left corner of page
trigger('mouseup')
```
It mimics the behavior of dragging mouse from content to outside of content. But unlike browser, cypress will not trigger an automatic click event after that. 

And if I chain it with a `.trigger('click')`, the test would be testing "clicking outside of dialog closes it". And it lost the purpose of testing the automatic click triggered after dragging.

Reply:
Ahaa it worked ❤️ on cypress with my change chained 'click' does not close dialog